a2-synchrotron-contiki/core/net/mac/chaos/service/cluster/cluster.h

```diff
@@ -0,0 +1,13 @@
+#ifndef _CLUSTER_H
+#define _CLUSTER_H
+
+#include "chaos-control.h"
+
+extern const chaos_app_t cluster;
+extern uint8_t is_cluster_service_running;
+
+#ifndef NODE_LIST_LEN
+#define NODE_LIST_LEN 10  //describes how many nodes can become clusterheads.
```

Comment:
Nodes should remember if they've been assigned as cluster heads in previous rounds. in order to let more candidates be elected.

Reply:
Trello card

Reply:
Added